keyboards/handwired/marek128b/ergosplit44/rules.mk

```diff
@@ -0,0 +1,2 @@
+# This file intentionally left blank
+SRC += rgb.c
```

Comment:
```suggestion
```

Reply:
"or the code is modified appropriately to live at the keymap level."